R/Fit_mult_impute_observed_outcome.R

```diff
@@ -58,7 +57,10 @@ fit_mult_impute_obs_outcome <- function(mids,
 
 #' Remove missing cases from a mids object
 #'
-#' Remove_missing_from_mids is used to filter a mids object for missing cases
+#' !!! Deprecated !!! This used to be a helper function for basecamb::fit_mult_impute_obs_outcome
```

Comment:
Was there a reason why we made this a public function in the first place rather than keeping it internal? Is there much use that cannot be replicated with `apply_function_to_imputed_data()`?

Removing or hiding the function without a transition period is not great style. To properly deprecate the function, leave the namespace and functionality unaffected and use the `.Deprecated` function (https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/Deprecated).
```suggestion
#' .Deprecated(new='apply_function_to_imputed_data', package='basecamb')
```

That call should be the first call in the function body. Feel free to hint at deprecation in the description, but a simple notice should do. the `.Deprecated` call automatically issues a message whenever the function is used.

Reply:
agreed